includes/provider-string-format.md

```diff
@@ -0,0 +1,4 @@
+ 
+However, when calling the **String.Format** method, it is not necessary to focus on the particular overload that you want to call. Instead, you can call the method with an object that provides culture-sensitive or custom formatting and a [composite format string](~/docs/standard/base-types/composite-formatting.md) that includes one or more format items. You assign each format item a numeric index; the first index starts at 0. In addition to the initial string, your method call should have as many additional arguments as it has index values. For example, a string whose format items have indexes of 0 and 1 should have 2 arguments; one with indexes 0 through 5 should have 6 arguments. Your language compiler will then resolve your method call to a particular overload of the **String.Format** method.   
```

Comment:
There are performance considerations with respect to boxing here. If you call an overload of `string.Format` with a set of arguments of different types, that will likely resolve to the overload that takes a `params object[]`. If any of those types are value types, they will be boxed.

Should that be mentioned here and in the simple-string-format?